Regression in 3.53.0 caused by small buffer for paths
TODO: Will be fixed in the next release, remove then
diff --git a/src/shell.c.in b/src/shell.c.in
index b121ed8..d1d7a5e 100644
--- a/src/shell.c.in
+++ b/src/shell.c.in
@@ -12316,7 +12316,9 @@ static int runOneSqlLine(
   rc = shell_exec(p, zSql, &zErrMsg);
   END_TIMER(p);
   if( rc || zErrMsg ){
-    char zPrefix[100];
+    char zPrefix[2048
+                 /* must be relatively large:
+                 ** https://sqlite.org/forum/forumpost/205f73db1b2806f5 */];
     const char *zErrorTail;
     const char *zErrorType;
     if( zErrMsg==0 ){
